### DISCUSSION TITLE
WinUI3 0.5 ( Desktop C++ ) for Win8 Win7

### DISCUSSION THREAD

**Discussion by r2156master** [microsoft/WindowsAppSDK#877] [Q&A] [answered]
Hello, i would like to know if its possible to run win32 apps made with WinUi3 0.5 on windows 7 windows 8 ?

And Is it normal that XAML design isn't working for C++ winui projects ?

**Comment by mveril** [comment, marked as answer]
Hello
From the [documentation](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/)

> WinUI 3 - Project Reunion 0.5 is compatible with PCs running the Windows 10 October 2018 Update (version 1809 - build 17763) and newer.

Maybe I'm wrong but I think it's going to stay that way.

For the XAML designer I don't know.

**Reply by r2156master** [reply]
Okay thanks,  i also found compatibility table https://microsoft.github.io/microsoft-ui-xaml/about.html
that's actually sad win7 win8 are pretty usable os

**Comment by jaigak** [comment]
Windows 7, Windows 8 and Windows 8.1 RTM is no longer supported and very few users use Windows 8.1 Update 1.
